src/components/pages/books/books.css

```diff
@@ -185,3 +185,41 @@
 .addBtn:hover {
   cursor: pointer;
 }
+
+@media screen and (max-width: 768px) {
+  .book-section-one {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .book-title {
```

Comment:
| Zeplin's Design| @AleWaweru Design |
|-|-|
| ![screenshot](https://github.com/AleWaweru/Bookstore/assets/93039359/c6f4c65c-d3f5-44e0-9698-6e323de24e52) | ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/83978ef6-ab3b-4832-854a-68da4ecd0b34) |

- [ ] It would be nice if you make your contents adjust as you can see in the Zeplin's design to make your design more professional and attractive 👍

Reply:
done